### ISSUE TITLE
Error fetching images in 1.11 releases

### ISSUE THREAD

**Issue by batmike** [algolia/algoliasearch-magento-2#818]
I've tested the 1.11 release series with our Commerce install version 2.3.1 and 2.3.2. Each time, the resulting index in Algolia contains only the placeholder images.

All of our configurable products do not have any images assigned so everything is pulled from the child simple products. However, something is not working right in the 1.11 series as it just gets the placeholders and that's all that ends up showing on the InstantSearch pages and the auto-complete.

Let me know if more information would help in troubleshooting this. Thanks!

**Comment by bsuravech** [comment]
Hi @batmike! Can you confirm that this was working in the 1.10 version? Was there any customizations made on your end for the previous version to pull child simple product images to its parent? Perhaps a backend observer on product indexing? 

The extension does have a feature where you can switch the color variation image from it's simple product. This is a configuration that can be found in Configuration > Algolia Search > Products. Would this help your issue?

**Comment by batmike** [comment]
@bsuravech - It is working as expected in the 1.10 release. I’ve also checked and the setting for replacing the image with the variation is set to Yes and has been in both 1.10 and in our testing of 1.11.

Other than that, we do not have any custom observers or anything else that would be interfering with the processing of the images as the indexes are built.

**Comment by bsuravech** [comment]
Hi @batmike,

I see it now. We have PR#727 had updated the product image helper to pull the correct product image type, however, it seems that $this->getProduct()->getImage() that was removed by this was actually setting the child image as the parent's image. 

https://github.com/algolia/algoliasearch-magento-2/pull/727

We have created a ticket internally to re-add the ability to get the correct image and type from the child product if the parent does not have an image assigned. 

We will update once this issue once it has been addressed. Thank you for reporting!
